packages/big-design/src/components/FileUploader/DropZone.tsx

```diff
@@ -1,8 +1,5 @@
-import {
-  DraftIcon,
-  FileDownloadIcon,
-  RemoveCircleOutlineIcon,
-} from '@bigcommerce/big-design-icons';
+import { FileDownloadIcon, RemoveCircleOutlineIcon } from '@bigcommerce/big-design-icons';
+import * as icons from '@bigcommerce/big-design-icons';
```

Comment:
This ends up importing all the icons which prevents them from being tree-shaked. Since that increases the bundle size for this component, I recommend just having a prop that consumes an icon instead (see [`Button` component](https://github.com/bigcommerce/big-design/blob/main/packages/big-design/src/components/Button/Button.tsx#L15)).

Reply:
Done